torchci/lib/fetchFlakyTests.ts

```diff
@@ -8,7 +8,7 @@ export default async function fetchFlakyTests(num_hours: string): Promise<FlakyT
     rocksetClient.queryLambdas.executeQueryLambda(
       "commons",
       "flaky_test_query",
-      "12b03f2fbf584dc0",
+      "7e7c838ec2592c60",
```

Comment:
this is a query change that adds head_branch as a field to return: 
```
SELECT 
  flaky_tests.name,
  flaky_tests.suite,
  flaky_tests.file,
  flaky_tests._event_time,
  sum(flaky_tests.num_green) AS "num_green",
  sum(flaky_tests.num_red) AS "num_red",
  ARRAY_AGG(flaky_tests.workflow_id) AS workflow_ids,
  ARRAY_AGG(workflow.name) as workflow_names,
>>>>>>>>>>>>>>
<<<<<<<<<<<<<< INSERTED
  ARRAY_AGG(workflow.head_branch) as branches,
>>>>>>>>>>>>>>
FROM commons.flaky_tests flaky_tests JOIN commons.workflow_run workflow on CAST(flaky_tests.workflow_id as int) = workflow.id
WHERE flaky_tests._event_time > (CURRENT_TIMESTAMP() - HOURs(:num_hours))
GROUP BY name, suite, file, _event_time
```

Reply:
do we do anything with the flaky tests in other branches? If not, could we filter in the query itself?